lib/vorpal.js

```diff
@@ -97,7 +97,7 @@ function Vorpal() {
   });
 
   // Allow unix-like key value pair normalization to be turned off by toggling this switch on.
-  this.isCommandArgKeyPairNormalized = true;
+  this.isCommandArgKeyPairNormalized = false;
```

Comment:
There's no way of altering this with an option passed when initializing a Vorpal instance?

Reply:
It does not seem to be intended. Even though we could add this, since we do not have the positive case planned (using unix-like key value pairs) - it does not seem to be necessary for this PR.